frontend/src/routes/Applications/AdvancedConfiguration.tsx

```diff
@@ -12,41 +17,120 @@ import {
     TextContent,
     Text,
     TextVariants,
+    ToggleGroup,
+    ToggleGroupItem,
 } from '@patternfly/react-core'
 import { ExternalLinkAltIcon } from '@patternfly/react-icons'
 import { DOC_LINKS } from '../../lib/doc-util'
 import { useCallback, useMemo } from 'react'
+import queryString from 'query-string'
+import { useHistory } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
 import { useRecoilState } from 'recoil'
-import { applicationsState } from '../../atoms'
+import { channelsState, placementRulesState, placementsState, subscriptionsState } from '../../atoms'
 import { IResource } from '../../resources'
+import _ from 'lodash'
 
 export default function AdvancedConfiguration() {
-    const { t } = useTranslation()
+    const { t } = useTranslation('application')
```

Comment:
Not sure if this application would affect any translation work.

Reply:
I think the idea is to have a single file with translations (ie. not specify 'applications' here). The keys are just the english strings themselves. So then later, there is a tool to run to collect these into a single file. @RobDolares @jamestalton please correct me if I'm wrong.

Reply:
@KevinFCormier @chenz4027 - That is correct. we won't reference a specific translation file.  Instead the full text needs to be in the file where it will be translated.  We are still not sure how dynamic string injection is going to be handled, but in cases where enums are used for a dynamic string, i have been creating `switch` statements to return all possible values until we can determine best way to handle string interpolation. @jamestalton

Reply:
Thanks Robert, I'll make the change accordingly.